Args.cs

```diff
@@ -0,0 +1,91 @@
+﻿using System;
+
+namespace dropdownloadcore
+{
+    public class Args
+    {
+        public string VstsPat { get; set; }
+
+        public string DropDestination { get; set; }
+
+        public string DropUrl { get; set; }
+
+        public string RelativePath { get; set; }
+
+        public void Parse(string[] args)

```

Comment:
any reason you didn't just use. https://www.nuget.org/packages/CommandLineParser/